src/citrine/resources/team.py

```diff
@@ -99,37 +101,81 @@ def list_members(self) -> List[TeamMember]:
                                             version=self._api_version)["users"]
         return [TeamMember(user=User.build(m), team=self, actions=m["actions"]) for m in members]
 
-    def remove_user(self, user_id: Union[str, UUID]) -> bool:
+    def get_member(self, user_id: Union[str, UUID, User]) -> TeamMember:
```

Comment:
Informational question: are you thinking we should start to introduce this pattern across the SDK? Because there are a bunch of places we accept the UUID, but not the object itself.

Reply:
There are also a number of places where we use this pattern.  I've been slowly migrating over to it when I touch files.

Reply:
Gotcha, I'll start doing the same.